packages/common/container-definitions/src/loader.ts

```diff
@@ -328,7 +328,7 @@ export interface IContainer extends IEventProvider<IContainerEvents> {
 
 	/**
 	 * Represents the resolved url to the Container.
-	 * Will be undefined only when the container is in the {@link AttachState.Detached | detatched} state.
+	 * Will be undefined only when the container is in the {@link (AttachState:variable).Detached | detatched} state.
```

Comment:
```suggestion
	 * Will be undefined only when the container is in the {@link (AttachState:variable).Detached | detached} state.
```
The h-less Iron Lady would like a word 😝 .